.github/workflows/test-reproduction-run.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Full Run
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  test-full-run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements.txt
+
+    - name: Run with reproduction data
+      run: |
+        ./run map -r tests/data/ci-run -t $(date +%s)
```

Comment:
As mentioned in the other message, I would prefer if we would run a reproduction run with the actual static timestamp and then evaluate the hash to be what we expect. With the dynamic date the hash wouldn't be deterministic and I think we might be missing on some potential coverage of our deteriminism.